src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js

```diff
@@ -211,6 +213,39 @@ export const BulkEditListFilters = ({
     }
   }, [location.search]);
 
+
+  const getisDropzoneDisabled = (capabilitie) => {
+    switch (capabilitie) {
+      case CAPABILITIES.USER:
+        if (hasOnlyViewCsvPerms) {
+          return true;
+        } else return isDropZoneDisabledPerm;
+      case CAPABILITIES.ITEM:
+      case CAPABILITIES.HOLDING:
+        if (hasViewInAppPerms) {
+          return true;
+        } else return isDropZoneDisabledPerm;
+
+      default: return isDropZoneDisabledPerm;
+    }
+  };
+
+  const getIsSelectIdentifiersDisable = (capabilitie) => {
+    switch (capabilitie) {
+      case CAPABILITIES.USER:
+        if (hasOnlyViewCsvPerms) {
+          return true;
+        } else return isSelectIdentifiersDisabled;
+      case CAPABILITIES.ITEM:
+      case CAPABILITIES.HOLDING:
+        if (hasViewInAppPerms) {
+          return true;
+        } else return isSelectIdentifiersDisabled;
+
+      default: return isSelectIdentifiersDisabled;
+    }
+  };
+
```

Comment:
1. Logic and implementation on both the above methods seems to be same. It is better to use a single one for both the implementations (probably by accepting another param like 'type' which would say select of dropZone).
2. Also, the implementation should be moved to "src/components/BulkEditList/BulkEditListFilters/utils".
3. Add screenshots or recordings of the behavior.

Reply:
@Terala-Priyanka Great idea, I also thought in that way. I can't add video recording because it's only permission check function.

Reply:
@Terala-Priyanka I have updated changes, please take a look. Also added tests for function